keyboards/keychron/q11/info.json

```diff
@@ -0,0 +1,287 @@
+{
+    "keyboard_name": "Keychron Q11",
+    "manufacturer": "Keychron",
+    "url": "https://github.com/Keychron",
+    "maintainer": "lalalademaxiya1",
+    "processor": "STM32L432",
+    "bootloader": "stm32-dfu",
+    "usb": {
+        "vid": "0x3434"
+    },
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "dip_switch": true,
+        "extrakey": true,
+        "mousekey": true,
+        "encoder": true,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "matrix_pins": {
+        "cols": ["A7", "A6", "A5", "A4", "A3", "A2", "A1", "C15", null],
+        "rows": ["A13", "A14", "A15", "B3", "B4", "B5"]
+    },
+    "diode_direction": "ROW2COL",
+    "split": {
+        "enabled": true,
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "C14", "pin_b": "C15"}
+                ]
+            }
+        },
+        "matrix_pins": {
+            "right": {
+                "cols": ["A8", "A7", "A6", "A5", "A4", "A3", "A2", "A1", "A0"],
+                "rows": ["B5", "B4", "B3", "A15", "A14", "A13"]
+            }
+        },
+        "transport": {
+            "protocol": "serial_usart",
+            "sync_matrix_state": false
```

Comment:
if these have per-key rgb, then you probably want to enable this, TBH.  Otherwise, reactive modes won't trigger on the slave/secondary side.

Reply:
Thanks for your kind suggestions, it’s been updated.